### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by mikolak**
I'll later merge master into develop.

## 2.1.0

* **BREAKING** ScanResult.AdvertisementData.ManufacturerData has changed type from Int8List to Uint8List
* Fix issue with invalid value of manufacturer data in scan results
* Fix issue with initialising plugin from a service without an active activity
* Update README with information about permissions
* Fix issue with default value of MTU when connecting to a peripheral on Android
* Fix issue where first few notifications right after start of monitoring a characteristic could get lost

### CODE REVIEW

File: android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public static void registerWith(Registrar registrar) {
         final EventChannel connectionStateChannel = new EventChannel(registrar.messenger(), ChannelName.CONNECTION_STATE_CHANGE_EVENTS);
         final EventChannel characteristicMonitorChannel = new EventChannel(registrar.messenger(), ChannelName.MONITOR_CHARACTERISTIC);
 
-        final FlutterBleLibPlugin plugin = new FlutterBleLibPlugin(registrar.activity().getApplicationContext());
+        final FlutterBleLibPlugin plugin = new FlutterBleLibPlugin(registrar.context());
```
Comment (dariuszseweryn) [inline android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java]: Why not `getApplicationContext()`?

Comment (mikolak) [reply android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java]: This is, in fact, application context.
https://api.flutter.dev/javadoc/io/flutter/plugin/common/PluginRegistry.Registrar.html#context--

Comment (dariuszseweryn) [reply android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java]: We are not yet up to the new v2 API I suppose?

Comment (bartoszwilk) [reply android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java]: Maybe it's a good idea to migrate android plugin to v2 as it's a minor release? What do you guys think?

Comment (dariuszseweryn) [reply android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java]: We can do that in `2.2.0` no worries ;)

**Pull request merged**